plugins/router/f5/f5.go

```diff
@@ -307,15 +318,24 @@ func newF5LTM(cfg f5LTMCfg) (*f5LTM, error) {
 		privkeyFileName = newPrivkeyFile.Name()
 	}
 
+	partitionPath := F5DefaultPartitionPath
+	if len(cfg.partitionPath) > 0 {
+		partitionPath = cfg.partitionPath
+	}
```

Comment:
Well, I guess am looking at this as a standalone unit - this code could be used from anywhere, so if the partitionPath is not set, use a the default partition path, otherwise use what was specified. It also handles the case when partition path is set to an empty string.